engine/src/test/java/io/zeebe/engine/util/TestStreams.java

```diff
@@ -129,8 +129,13 @@ public LogStream createLogStream(final String name, final int partitionId) {
       throw new UncheckedIOException(e);
     }
 
-    final AtomixLogStorageRule logStorageRule = new AtomixLogStorageRule(dataDirectory, partitionId);
-    logStorageRule.open(b -> b.withDirectory(segments));
+    final AtomixLogStorageRule logStorageRule =
+        new AtomixLogStorageRule(dataDirectory, partitionId);
+    logStorageRule.open(
+        b ->
+            b.withDirectory(segments)
+                .withMaxEntrySize(4 * 1024 * 1024)
```

Comment:
Do we need to set this?

Reply:
I ran into some tests which expect to store large events of 4mbs :man_shrugging:

Reply:
But could'nt the test then configure that? I mean now this is default for all tests, wouldn't this increase the footprint and decrease the performance of our tests?

Reply:
Considering the TestStreams interface, configuring this is rather indirect, but we can do it.